src/app/components/crypto-assets/choose-crypto-asset/choose-crypto-asset.layout.tsx

```diff
@@ -13,9 +17,9 @@ export function ChooseCryptoAssetLayout({ children }: StackProps) {
     >
       <Box pb="space.05" pt={['unset', 'space.05']} px="space.05">
         <styled.h1 textStyle="heading.03">
-          CHOOSE ASSET
+          {title1}
           <br />
-          TO SEND
+          {title2}
```

Comment:
is there another way we can do this without the `<br />` weird breaking into two props